lib/burp/html/issue.rb

```diff
@@ -84,6 +84,18 @@ def method_missing(method, *args)
       method_names = translations_table.fetch(method, method.to_s)
       method_names = [method_names].flatten
 
+      # Process the Location field
+      if method.to_s == 'location'
+        location = @html.at_xpath('//span[contains(@class, "BODH1")]')&.text
+
+        if location
+          # Remove the header number
+          return location.split(/[[:space:]]/).drop(1).join(' ')
```

Comment:
@aapomm Can you add a comment here showing the example? so that it's clear what this method is trying to do.
```suggestion
# E.g. <span class="BODH1" id="1.0">1.0.&nbsp;http://10.0.0.1/location</span>
```